packages/Rakefile

```diff
@@ -42,6 +42,8 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
   end
 
   def define_archive_task
+    source_archive_url = built_package_url(nil, @archive_base_name)
+    download(source_archive_url, "..")
```

Comment:
If we write this here, this is always processed. (`rake --help` also processes this.)
We should put this into `file do ... end`.

Reply:
Fixed.